org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/translator/ScopeComputer.java

```diff
@@ -66,6 +67,8 @@
  * <p>
  * Please see ScopeComputer.java for the exact rules for deriving the missing
  * scopes.
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
store the trace scope (min and max scope); also, some errors on scopes are relaxed: eg, `one var` sigs may still have a scope larger than 1, since the multiplicity is imposed state by state, thus the atom may change